### ISSUE TITLE
User survey translations - tracking issue

### ISSUE THREAD

**Issue by InessaPawson** [scikit-learn/administrative#61]
If you are available to help the sk-learn survey team with the questionnaire translations, please mention what language you can help us with in a comment below.

**Comment by betatim** [comment]
I speak Deutsch so could help with translating to German.

**Comment by norbusan** [comment]
de: native
jp: business level
it: fluent
en: obvious

**Comment by InessaPawson** [comment]
@betatim We are not planning on translating this survey into German. I truly appreciate you raising your hand to help and will keep your offer in mind for future needs.

**Comment by InessaPawson** [comment]
@norbusan We would love to translate the survey questionnaire into Japanese. Do you think you could recruit another volunteer with knowledge of Japanese for the backward translation?

**Comment by tkoyama010** [comment]
I may late for party but I can translate Japanese for native level.

**Comment by norbusan** [comment]
@InessaPawson @tkoyama010 Thanks for the reaction. I can surely translate with colleagues here, but of course I am not native, so if Koyama-san can take this task, it will be definitely much better executed than I can do.

**Comment by InessaPawson** [comment]
@tkoyama010 Tetsu, thank you so much for your willingness to help. I’m excited to collaborate with you on another translation that will help the Japanese scientific Python community to voice their thoughts on the development of the ecosystem.

@norbusan Since we do need 2 translators. Would you be available to do the backward-translation (from Japanese into English)?

**Comment by norbusan** [comment]
@InessaPawson thanks to both of you.
Yes I am fine with doing the backward translation!

**Comment by NoorhanAbbas** [comment]
Hello, I speak Arabic and can help with the translation.

**Comment by alexdesiqueira** [comment]
Hey @InessaPawson,
I can help with the Portuguese translation.

**Comment by InessaPawson** [comment]
@NoorhanAbbas Thank you for raising your hand! We really hoped to provide an Arabic version of the questionnaire. 
Please email me to inessapawson@gmail.com from the email address that is best for the communication related to this task.

**Comment by InessaPawson** [comment]
@alexdesiqueira Thank you, Alex! Excited to collaborate on another translation with you to elevate the voices of the Portuguese speaking community.

**Comment by InessaPawson** [comment]
@norbusan Is the email address listed in your GH profile best for communication related to this task? If not, please email me to inessapawson@gmail.com from the email address that is best for it.

**Comment by norbusan** [comment]
> @norbusan Is the email address listed in your GH profile best for communication related to this task? 

Yes, don't worry, I got your email and will check the translation asap, probably today.

**Comment by razekmh** [comment]
@InessaPawson Arabic here too, If needed

**Comment by InessaPawson** [comment]
@norbusan Thank you for confirming receiving my email! Just wanted to make sure it’s not buried in dozens/hundreds of GitHub notifications that you might be receiving to that address.

**Comment by InessaPawson** [comment]
@razekmh Awesome! I will forward you the Arabic translation for the backward translation once it's ready.

**Comment by InessaPawson** [comment]
@razekmh The Arabic translation is ready for your review. Please email me to inessapawson@gmail.com from the email address that is best for the communication related to this task.

**Comment by razekmh** [comment]
@InessaPawson done ✅

**Comment by razekmh** [comment]
@NoorhanAbbas @InessaPawson 
This is a comment on the Arabic translation methodology. Please feel free to move elsewhere if needed. 

> The following general guidelines should be considered in this process:
> - Translators should always aim at the conceptual equivalent of a word or phrase,
not a word-for-word translation, i.e. not a literal translation. They should consider the definition of the original term and attempt to translate it in the most relevant way.

from "sk-learn user survey - translation considerations"

Many of the technical terms used in the survey are used by Arabic speaking practitioners in their original English. In making the survey usable by an Arabic speaker we have essentially three options to represent any term (let's consider the term `regression` for example) :
- Use the English original phrase `regression`
- Use transliteration `ريجريشن`
- Use Arabic translation `انحدار`

Each has advantages and disadvantages. Using the original English is not much of a translation, transliteration might be tricky to reverse and the Arabic term might sound alien to many. We could use a combination of the three by including both the original English next to the Arabic translation or transliteration `regression/انحدار` which will cover a broader base of users. This introduces its own difficulties with aligning the mixed Latin/Arabic scripts correctly, but I believe is the better solution. When translating a Term to Arabic, there could be a few words that fit the general meaning of the term. However, for consistency, I prefer to use a reference. I recommend the [glossary](https://sdaia.gov.sa/ar/MediaCenter/KnowledgeCenter/ResearchLibrary/SDAIAPublications15.pdf) published by the Saudi Data & AI Authority.

**Comment by NoorhanAbbas** [comment]
@razekmh @InessaPawson Thank you very much for your efforts to make the translation better.  I agree with your comments and support your recommendation.  It has been a pleasure working with you!  All the best

**Comment by reshamas** [comment]
@InessaPawson Is this issue ready to be closed?
